src/display/font_loader.js

```diff
@@ -354,7 +354,7 @@ class FontFaceObject {
       fontRegistry = null,
     }
   ) {
-    this.compiledGlyphs = Object.create(null);
```

Comment:
`Object.create(null)` is used on purpose to get an object without prototype, so please keep that.

Reply:
I think it doesn't matter performance-wise (since many fonts won't be created... or is there some other reason for doing this?), but I reverted since I changed the rest of the code to use a way for `hasOwnProperty` to not be necessary.

Reply:
One of the reasons this is done is because PDF.js is often used in combination with other libraries which sometimes alter/pollute the object's prototype, which has caused bugs before. Even though it's those libraries' fault for polluting the object's prototype, PDF.js is sometimes blamed for it since "it doesn't work with library X". We don't need the prototype in these cases, so it's best not to use it to avoid problems.

Reply:
I see and makes sense. That sucks though haha